imsv-docs-docusaurus/openapi/endpoints/transactions/models/transaction.yaml

```
@@ -83,6 +83,18 @@ properties:
     enum:
       - "credit"
       - "debit"
+  paymentType:
+    type: string
+    description: The type of payment
+    enum:
+      - "purchase"
+      - "refund"
+      - "adjustment"
+    example: "purchase"
+  relatedPaymentId:
+    type: string
+    description: The ID of the related transaction, if applicable
+    example: "1000000178145"
```

This example is using an external payment id format.

```suggestion
    example: "b297658d0cbac11"
```